src/pages/java/abstract-class/index.md

```diff
@@ -4,38 +4,45 @@ title: Abstract Classes in Java
 
 Lets discuss abstract classes. Before diving into this tutorial it is better that you have understood concepts of classes
```

Comment:
Make this one complete sentence instead of adding a new line character in between.

Reply:
Sorry, I am a little confused as to where you are asking me to make a change. I did not touch this part and left it as it was from the previous commit. Are you telling me that I should remove the new line at 4 and move the content at line 5 to line 4?

Reply:
Yeah, moving the content from 4 to 5 but it seem resolved as of now.